lib/setup.js

```diff
@@ -125,7 +125,7 @@ function paths (opts) {
 	var ret;
 	
 	if (Array.isArray(opts.paths) && opts.paths.length > 0) {
-		ret = opts.paths.map(function (p) { return path.join(opts.cwd, p); });
+		ret = opts.paths.map(function (p) { return path.relative(opts.cwd, p); });
```

Comment:
This won't quite work for all of the other scenarios either. Going to test a more complete solution here to get that support back.